Command/FindContentCommand.php

```diff
@@ -13,6 +13,7 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputOption;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
```

Comment:
This should be:
`use eZ\Publish\API\Repository\Values\Content\Query;`


With that, line below `$query = new \eZ\Publish\API\Repository\Values\Content\Query();` can be changed to `$query = new Query();`